.github/workflows/release.yml

```diff
@@ -97,16 +98,15 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const { owner, repo } = context.repo
-            await github.repos.deleteRelease({ owner, repo, release_id: ${{ steps.release.outputs.release_id }} })
+            await github.repos.deleteRelease({ owner, repo, release_id: ${{ github.events.release.release_id }} })
```

Comment:
It looks like GitHub might allow both forms, but you're using "github.event" in some places and "github.events" other places. The documentation lists "github.event" so we probably should use this form everywhere just to be safe.

Reply:
Fixed, all should be `event` instead.